src/main/java/slimeknights/tconstruct/common/ClientProxy.java

```diff
@@ -24,6 +26,6 @@ public void debug() {
 
   @Override
   public void spawnSlimeParticle(World world, double x, double y, double z) {
-    mc.particles.addEffect(new SlimeFxParticle(world, x, y, z, new ItemStack(CommonItems.blue_slime_ball)));
+    mc.particles.addEffect(new SlimeFxParticle(world, x, y, z, new ItemStack(FoodItems.blue_slime_ball.get())));
```

Comment:
The get is unneeded, the whole point of ItemObject is it implements IItemProvider, so you can make an item stack from it.

Reply:
so nix the `.get()`? I was mostly thinking in terms of consistency.